apps/callscreen/js/calls_handler.js

```diff
@@ -845,12 +839,42 @@ var CallsHandler = (function callsHandler() {
     holdOrResumeSingleCall();
   }
 
+  /**
+   * Check if a call is being established.
+   *
+   * @returns true if a call is being established, false otherwise
+   */
   function isEstablishingCall() {
     return telephony.calls.some(function(call) {
       return call.state == 'dialing' || call.state == 'alerting';
     });
   }
 
+  /**
+   * Check if any of the calls is currently on hold.
+   *
+   * @returns true if a call is on hold, false otherwise
+   */
+  function isAnyCallOnHold() {
+    return telephony.calls.some((call) => call.state === 'held') ||
```

Comment:
FYI, you can exclude the ```()``` around passed parameters if there's only one parameter, like this:
```js
return telephony.calls.some(call => call.state === 'held') ||
```